### PR TITLE
chore(doc): fixed sri cheat-sheet readme

### PR DESCRIPTION
**Pull request by Mellywins**

### CODE REVIEW

File: docs/GL5/SRI/Indexation/cheat-sheet.md [hunk 1]
```diff
@@ -8,15 +8,15 @@ authors:
 
 Author [@medbensalah](https://github.com/medbensalah)
 
-## Indexation
+# Indexation
```
Comment (wadhah101) [inline docs/GL5/SRI/Indexation/cheat-sheet.md]: Only one H1 is allowed.
H1 must be the name of the document

File: docs/GL5/SRI/Indexation/cheat-sheet.md [hunk 2]
```diff
@@ -8,15 +8,15 @@ authors:
...
 
 ### Definition
 
 Processus permettant de construire un ensemble d'éléments
-**clés** permettant de caractériser le contenu d’un
+_clés_ permettant de caractériser le contenu d’un
```
Comment (wadhah101) [inline docs/GL5/SRI/Indexation/cheat-sheet.md:16]: **** is the official markdown for bold  
https://www.markdownguide.org/basic-syntax/#bold

File: docs/GL5/SRI/Indexation/cheat-sheet.md [hunk 3]
```diff
@@ -30,9 +30,9 @@ document / retrouver ce document en réponse à une requête
 - Vocabulaire contrôlé
 - Thésaurus
 
-#### Construction d'index
+#### Construction d'index<br>
```
Comment (wadhah101) [inline docs/GL5/SRI/Indexation/cheat-sheet.md]: Never use HTML in MD

File: docs/GL5/SRI/Indexation/cheat-sheet.md [hunk 4]
```diff
@@ -30,9 +30,9 @@ document / retrouver ce document en réponse à une requête
...
 
-![img](assets/Construction.png)
+![](./assets/Construction.png)
```
Comment (wadhah101) [inline docs/GL5/SRI/Indexation/cheat-sheet.md:35]: we need text inside [ ] as alt

File: docs/GL5/SRI/Indexation/cheat-sheet.md [hunk 5]
```diff
@@ -58,24 +58,25 @@ document / retrouver ce document en réponse à une requête
...
+
+- Recherche de groupes de mots
+  - n-grammes ; n > 1 / un n-gramme sera considéré comme un terme
 
 #### Construction de l'index
 
 - Algorithme naïf
 
-  ```algorithm
+  ```
```
Comment (wadhah101) [inline docs/GL5/SRI/Indexation/cheat-sheet.md:77]: we need to specify the language.
We can use Algorithm for non code blocks

File: docs/GL5/SRI/Indexation/cheat-sheet.md [hunk 6]
```diff
@@ -84,91 +85,75 @@ document / retrouver ce document en réponse à une requête
...
 
-  - Indexation
-  
-  - ![sc](assets/Screenshot_100.png) => ![sc](assets/Screenshot_101.png)
-  
-  - Compression
-  Minimiser le nombre de bits transférés
-
-### Variable-Byte Encoding
+    ![](./assets/Screenshot_99.png)
```
Comment (wadhah101) [inline docs/GL5/SRI/Indexation/cheat-sheet.md:127]: Same add alt

**Review by wadhah101 — CHANGES_REQUESTED** [review]

File: docs/GL5/SRI/Indexation/cheat-sheet.md (hunk 1 above)
Comment (Mellywins) [reply docs/GL5/SRI/Indexation/cheat-sheet.md]: LGTM 👍

**Comment by wadhah101** [comment]
Forked Fixed Merged 🔥🔥🔥

**Pull request merged**